packages/location-icon/src/index.tsx

```diff
@@ -37,29 +40,32 @@ export function LocationIcon({
         <S.FromIcon
           className={className}
           size={size}
-          title={title || "From Location Icon"}
+          title={title || intl.formatMessage({ id: "otpUi.LocationIcon.from" })}
```

Comment:
I think a better approach would be to have this icon not have any alt text or title for these reasons, and that the alt text should be determined by the components that use these icons:
- "From"/"To" text is taken out of context, and other languages could use other words (e.g. origin, destination).
- The title text is not shown to screen readers on the LocationField component (although a "From"/"To" tooltip is shown on hovering the button - that tooltip could be more descriptive instead)
- The title prop generates unnecessary extra text for the icon components that are placed on the map, and you'd have.

Reply:
For the "generic" or "intermediate" location icon, it looks like the description text we provide by default does not match the actual use with intermediate locations.

Reply:
I agree with you -- for the record, this is a departure from how it used to work, right?

Reply:
> for the record, this is a departure from how it used to work, right?

Correct.